bin/cylc-monitor

```diff
@@ -138,7 +141,8 @@ The USER_AT_HOST argument allows suite selection by 'cylc scan' output:
                     self.pclient.get_suite_state_summary())
             except Exception as exc:
                 print >> sys.stderr, "\033[1;37;41mERROR\033[0m", str(exc)
-                self.pclient.reset()
```

Comment:
(I suspect client reset use to have the desired effect - although I haven't looked back to check - but now it is just a stub)